src/Sulu/Bundle/MediaBundle/DependencyInjection/Configuration.php

```diff
@@ -54,6 +54,15 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
             ->end()
+            ->arrayNode('dropzone')
```

Comment:
Would you really put dropzone here? Can't we call it upload or something, who knows which other ways we will offer in the future.